src/static_shape.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright 2024 the Kurbo Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
+#![allow(unused_qualifications)]
```

Comment:
I didn't see a way to get this lint working, while leaving it enabled
it basically complains that we use `crate::$it` where `$it` evaluates to `Rect` because that is already in scope.
but the macro is called for other types which aren't in scope and for which the qualification isn't unnecessary.